plugins/com.google.cloud.tools.eclipse.login/META-INF/MANIFEST.MF

```diff
@@ -15,19 +15,19 @@ Require-Bundle: com.google.gson;bundle-version="[2.7,3.0)",
  org.eclipse.ui,
  org.mortbay.jetty.server,
  org.mortbay.jetty.util
-Bundle-Classpath: lib/com.google.cloud.tools.ide.login-0.0.0-SNAPSHOT.jar,
- lib/google-api-client-1.22.0.jar,
+Bundle-Classpath: lib/google-api-client-1.22.0.jar,
  lib/google-api-services-oauth2-v2-rev114-1.22.0.jar,
  lib/google-http-client-1.22.0.jar,
  lib/google-http-client-jackson-1.22.0.jar,
  lib/google-oauth-client-1.22.0.jar,
  lib/google-oauth-client-java6-1.22.0.jar,
  lib/google-oauth-client-jetty-1.22.0.jar,
+ lib/plugins-login-common-0.1.1-SNAPSHOT.jar,
  .
 Export-Package: com.google.api.client.auth.oauth2;x-friends:="com.google.cloud.tools.eclipse.appengine.deploy,com.google.cloud.tools.eclipse.appengine.deploy.ui",
  com.google.cloud.tools.eclipse.login,
  com.google.cloud.tools.eclipse.login.ui,
- com.google.cloud.tools.ide.login
+ com.google.cloud.tools.login
```

Comment:
Seems like the version recognized for this package is "0", probably because we didn't add this through OGSi. Are you suggesting we should add `com.google.cloud.tools.login;version="0"`?